tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/report.py

```diff
@@ -0,0 +1,40 @@
+import time
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base
+from gaiatest.form_controls.header import GaiaHeader
+from gaiatest.tests.functional.messages.test_sms_report import TestSmsReport
+
+class Report(Base):
+
+    _time_sent_locator = (By.CSS_SELECTOR, 'span.sent-timestamp')
+    _time_received_locator = (By.CSS_SELECTOR, 'span.received-timestamp')
+
+    _header_locator = (By.ID, 'information-report-header')
+
+    def __init__(self, marionette):
+        Base.__init__(self, marionette)
```

Comment:
In Python, if we didn't write these 2 lines, the results would be the same. We usually don't redefine a function if it just calls the parent function